app/src/main/java/org/mozilla/focus/utils/UrlUtils.java

```diff
@@ -183,4 +183,32 @@ public static String stripScheme(@Nullable String url) {
 
         return url.substring(start);
     }
+
+    public static String stripSchemeAndSubDomain(String url) {
```

Comment:
\[PMD\] INFO: The method 'stripSchemeAndSubDomain\(String\)' has a NCSS line count of 20\.